packages/components/asset/src/AssetIcon/AssetIcon.tsx

```diff
@@ -30,9 +31,10 @@ export function AssetIcon({
   className,
   ...otherProps
```

Comment:
There is a `AssetIcon.defaultProps` in the end of the file

Could you please move that default value here and remove that? 🙏 

I know it wasn't introduced with the PR, but It's just a quick tech debt win

Reply:
done